coalib/bears/Bear.py

```diff
@@ -99,7 +99,8 @@ class Bear(Printer, LogPrinter):
     and the CAN_DETECT sets. Possible values:
 
     >>> CAN_DETECT = {'Syntax', 'Formatting', 'Security', 'Complexity', 'Smell',
-    ... 'Redundancy', 'Simplification', 'Variable Misuse', 'Spelling', 'Other'}
+    ... 'Unused Code', 'Simplification', 'Variable Misuse', 'Spelling',
+    ... 'Memory Leak', 'Documentation', 'Duplication'}
```

Comment:
what is a ``double free`` and what would ``memory safety error`` do differently than ``memory leak``?